bahmni-lite/docker-compose.yml

```diff
@@ -279,6 +279,12 @@ services:
     # volumes:
     # - "${APPOINTMENTS_PATH:?}/dist/:/usr/local/apache2/htdocs/appointments"
 
+  sms-service:
+    image: bahmni/sms-service:${SMS_SERVICE_IMAGE_TAG:?}
+    profiles: ["sms"]
+    environment:
+      SMS_TOKEN: ${SMS_TOKEN:?}
```

Comment:
@kavitha-sundararajan Add the environment variable for ORIGINATOR as well

Reply:
added